integration/benchmark/cmd/manual/main.go

```diff
@@ -120,6 +160,11 @@ func main() {
 	statsLogger.Start()
 	defer statsLogger.Stop()
 
+	log.Info().
+		Int("number_of_accounts", int(maxTPS)**accountMultiplierFlag).
```

Comment:
```suggestion
		Int("number_of_accounts", int(maxTPS)*(*accountMultiplierFlag)).
```
Mixing the pointer and multiplication operations is a bit confusing for me